common/azure-common/src/main/java/com/azure/common/logging/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Package containing logging abstractions.
+ */
+package com.azure.common.logging;
```

Comment:
I don't think we need this logging infrastructure to be part of the public API - it will only be used by us.

Reply:
Moved to implementation, restructured code hierarchy.